.github/workflows/stale.yml

```diff
@@ -0,0 +1,37 @@
+# name: Close stale issues
+#
+# on:
+#   schedule:
+#     - cron: "30 12 * * *"
+#
+# jobs:
+#   stale:
+#     uses: hetznercloud/.github/.github/workflows/stale.yml@main
+
+name: Close stale issues
+
+on:
+  workflow_call:
```

Comment:
Do we need to explicitly pass the `GITHUB_TOKEN` like we did before?

Reply:
We do not need to pass the GITHUB_TOKEN manually, but we must ensure that we have the appropriate permissions in both the caller and called workflows.

Otherwise Github Action will fail:
```
Invalid workflow file: .github/workflows/stale.yml#L7
The workflow is not valid. .github/workflows/stale.yml (Line: 7, Col: 3): Error calling workflow 'hetznercloud/.github/.github/workflows/stale.yml@stale_bot'. The workflow is requesting 'issues: write, pull-requests: write', but is only allowed 'issues: none, pull-requests: none'.

```